src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GAPbaDetails.java

```diff
@@ -17,19 +17,19 @@
 public class GAPbaDetails {
 
     private final DynamicList applicantsPbaAccounts;
-    private final String pbaReference;
+    private final String serviceReqReference;
     private final Fee fee;
     private final PaymentDetails paymentDetails;
     private final LocalDateTime paymentSuccessfulDate;
 
     @JsonCreator
     GAPbaDetails(@JsonProperty("applicantsPbaAccounts") DynamicList applicantsPbaAccounts,
-                 @JsonProperty("pbaReference") String pbaReference,
+                 @JsonProperty("pbaReference") String serviceReqReference,
```

Comment:
Is corresponding field changed in CCD? If not i would suggest to keep it in sync as it gets difficult to track field in pojo against field in ccd.... Also if the field in ccd is changed then this piece of code will not work as there will not be any json property pbaReference

Reply:
I have not changed in ccd you said that will not work as there will not be any json property with pbaReference

Reply:
pbaReference is captured on screen... so I am trying to understand why we are overriding it with service reference?

Reply:
also for traceability it is better to have same name in pojo and in CCD

Reply:
Yes, added a new field serviceReqReference in GAPbaDetails  in General application and in General ccd  and retained Pbareference as it was